.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,46 @@
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+| Name  | about |title|labels|
+| ------------- | ------------- |------------- |------------- |
+| Bug Names  | Create a report to help us improve stellar-core |[Short Description] (Version: [stellar-core version])|bug|
```

Comment:
Can you revise the `Version: [stellar-core version]` and `Create a report to help us improve stellar-core` as all these are defined wrong?